### DISCUSSION TITLE
Set different wallpapers

### DISCUSSION THREAD

**Discussion by cyb3rgh0u1** [hyprwm/Hyprland#8565] [Q&A] [answered]
I recently started using Hyprland, and I was wondering if it's possible to set different wallpapers for each workspace. If yes, how can I configure it? Any help would be greatly appreciated!

**Comment by xaicat** [comment, marked as answer]
## Solution:

Yes, you can set different wallpapers for each workspace in Hyprland! Here's how to do it:

1. **Install `hyprpaper`:** 
Make sure you have hyprpaper installed, as it manages wallpapers in Hyprland. You can install it via AUR or your package manager:

`yay -S hyprpaper`


**Modify the Hyprland configuration file:**
Open your `~/.config/hypr/hyprland.conf` and add the following under the wallpaper section:

`hyprpaper=workspace1,~/Pictures/wallpaper1.jpg`
`hyprpaper=workspace2,~/Pictures/wallpaper2.jpg` 
`hyprpaper=workspace3,~/Pictures/wallpaper3.jpg`

Replace `workspace1, workspace2, etc.`, with the actual workspace numbers or names, and provide the correct paths to your wallpapers.

**Restart Hyprland or reload the config:**
After saving the changes, restart Hyprland or reload the config by running:

`hyprctl reload`

**Verify the changes:**
Switch between workspaces to ensure each one displays its assigned wallpaper.

**Optional Tips:**

Make sure your wallpapers are in supported formats `(e.g., .jpg, .png)`
If you're unsure of your workspace names, run hyprctl workspaces to list them.

**Reply by BadDev8** [reply]
Is there and updated way to do the same in config.lua?

**Reply by wabarTM** [reply]
@BadDev8 
1 - go read wiki
2 - stop necroposting
